build/publishNewVersion.mjs

```diff
@@ -92,22 +92,22 @@ const outputProcess = (process) => {
 
             const versionTag = `${VERSION_PREFIX}${newVersion}`;
             if (!options.dry) {
-                outputProcess(gitCommit(`chore(release): publish version ${versionTag} [version bump]`, '.'));
-                gitTag(versionTag);
+                outputProcess(await gitCommit(`chore(release): publish version ${versionTag} [version bump]`, '.'));
```

Comment:
Remove the outputProcess function, since the command don't return the process it's not useful anymore
```suggestion
                await gitCommit(`chore(release): publish version ${versionTag} [version bump]`, '.');
```